vagga.yaml

```diff
@@ -51,6 +51,18 @@ containers:
       /run: !Tmpfs
         subdirs:
           postgresql: { mode: 0o777 }
+  test:
+    setup:
+      - !Ubuntu xenial
+      - !PipConfig
+        dependencies: true
+      - !BuildDeps
+        - git
+      - !Depends .pre-commit-config.yaml
+      - !Py3Requirements requirements-test.txt
+      - !Install
```

Comment:
Please move this section up to `!BuildDeps` to avoid situation of installing python packages using `git+git://` URL when `git` is not installed yet.